src/libstd/fs.rs

```diff
@@ -482,23 +482,45 @@ impl File {
         self.inner.file_attr().map(Metadata)
     }
 
-    /// Creates a new independently owned handle to the underlying file.
+    /// Create a new `File` that shares the existing underlying file handle.
     ///
-    /// The returned `File` is a reference to the same state that this object
-    /// references. Both handles will read and write with the same cursor
-    /// position.
+    /// Modifying one `File` (e.g. reading, writing, seeking) will result in the
+    /// underlying file handle of the other `File` also adjusting.
```

Comment:
not a big fan of my wording here or in the previous section. lemme know if y'all have better suggestions

Reply:
Maybe something along the lines of:

> This means that both `File` instance shares the same cursor; reads, writes and seeks will affect both instances simultaneously.

Reply:
Thanks for the suggestion! Latest force push incorporates some of that wording